src/pykeen/nn/representation.py

```diff
@@ -175,6 +177,21 @@ def reset_parameters(self) -> None:
     def post_parameter_update(self):
         """Apply constraints which should not be included in gradients."""
 
+    def iter_extra_repr(self) -> Iterable[str]:
+        """Iterate over components for :meth:`extra_repr`."""
+        yield f"max_id={self.max_id}"
+        yield f"shape={self.shape}"
+        yield f"unique={self.unique}"
+        if self.normalizer is not None:
+            yield f"normalizer={self.normalizer}"
+        if self.regularizer is not None:
+            yield f"normalizer={self.regularizer}"
```

Comment:
typo

Reply:
https://github.com/pykeen/pykeen/pull/964/commits/7e2a6ffda4431b864b08666f4baf20cdf2436fc1

regularizer is a `nn.Module`, so we do not need to take care of having that included in the string representation